lighthouse-core/audits/dobetterweb/link-blocking-first-paint.js

```diff
@@ -50,7 +52,10 @@ class LinkBlockingFirstPaintAudit extends Audit {
   static computeAuditResultForTags(artifacts, tagFilter) {
     const artifact = artifacts.TagsBlockingFirstPaint;
 
-    const filtered = artifact.filter(item => item.tag.tagName === tagFilter);
+    const filtered = artifact.filter(item => {
+      return item.tag.tagName === tagFilter &&
+        (item.endTime - item.startTime) * 1000 >= LOAD_THRESHOLD_IN_MS;
```

Comment:
there might be varying opinions on this, but I'm in favor of not failing for <100ms since we're still notifying them of their existence in critical path

open to only loosening for blocking stylesheets (or a different threshold) and keeping all scripts or something though